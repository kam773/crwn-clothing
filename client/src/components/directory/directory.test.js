import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import { Directory } from './directory.component';

const mockProps = {
  sections: [
    {
      title: 'hats',
      imageUrl: 'https://i.ibb.co/cvpntL1/hats.png',
      id: 1,
      linkUrl: 'shop/hats',
    },
    {
      title: 'hats',
      imageUrl: 'https://i.ibb.co/cvpntL1/hats.png',
      id: 1,
      linkUrl: 'shop/hats',
    },
  ],
};
describe('create snapshot', () => {
  it('captures snapshot of Directory component', () => {
    const wrapper = renderer
      .create(<Directory {...mockProps} />)
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
