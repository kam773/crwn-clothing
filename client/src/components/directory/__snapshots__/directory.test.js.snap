// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot captures snapshot of Directory component 1`] = `
<div
  className="sc-EHOje gDbvGt"
>
  <div
    className="sc-bdVaJa HgIHW"
    onClick={[Function]}
  >
    <div
      className="sc-bwzfXH dEuOvA background-image"
    />
    <div
      className="sc-htpNat bxciRy content"
    >
      <span
        className="sc-bxivhb clMuku"
      >
        HATS
      </span>
      <span
        className="sc-ifAKCX dDPkuj"
      >
        SHOP NOW
      </span>
    </div>
  </div>
  <div
    className="sc-bdVaJa HgIHW"
    onClick={[Function]}
  >
    <div
      className="sc-bwzfXH dEuOvA background-image"
    />
    <div
      className="sc-htpNat bxciRy content"
    >
      <span
        className="sc-bxivhb clMuku"
      >
        HATS
      </span>
      <span
        className="sc-ifAKCX dDPkuj"
      >
        SHOP NOW
      </span>
    </div>
  </div>
</div>
`;
